includes/Blocks/Block.php

```diff
@@ -61,10 +61,23 @@ public function __construct( WP_Block_Type $block, Registry $block_registry ) {
 		$this->block_registry   = $block_registry;
 		$this->block_attributes = $this->block->attributes;
 		$this->type_name        = $this->format_type_name( $block->name );
-
+		$this->update_scalars();
```

Comment:
Also I would not put this method inside the block type as this gets called on every Block we register. Ideally you want to create a new class that inherits the  `OnInit` interface and call this inside the Registry `OnInit` method:

https://github.com/wpengine/wp-graphql-content-blocks/blob/main/includes/Registry/Registry.php#L52-L56